DependencyInjection/GrossumFaqExtension.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Grossum\FaqBundle\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+class GrossumFaqExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration();
+        $config        = $this->processConfiguration($configuration, $configs);
```

Comment:
@malavsky we should to decide about alignment.
I think it's a bad idea, because it's a very ugly in some cases like:
```
$superDuperLongVariable = 'value';
$myValue                = 'my value';
```

Also this long strings can cause line break, because line length is more than 120 chars.